libraries/botbuilder/tests/streaming/tokenResolver.test.js

```diff
@@ -183,6 +181,8 @@ describe(`TokenResolver`, function () {
     });
 
     it(`should get the token on the second try`, async function () {
+        this.timeout(10000);
+
```

Comment:
Note: Default timeout of 2000ms didn't pass. 5000ms passed on my machine, but made 10000ms to ensure this passes on CI.